### PR TITLE
First feature branch with migrations.

### PR DESCRIPTION
**Pull request by BrentFarese**

### CODE REVIEW

File: migrations/20170524131128-initial_migration.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        userId: {
+          type: Sequelize.UUID,
+          references: {
+            model: 'users',
+            key: 'id'
+          },
+          onDelete: 'SET NULL'
+        }
+      });
+
```
Comment (fugufish) [inline migrations/20170524131128-initial_migration.js:99]: We need to make sure to add indexes!

File: migrations/20170524131128-initial_migration.js [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        clientReference: {
+          type: Sequelize.STRING
+        },
+        legalType: {
+          type: Sequelize.STRING
+        },
+        importanceLevel: {
+          type: Sequelize.INTEGER
+        },
+        applicationId: {
```
Comment (fugufish) [inline migrations/20170524131128-initial_migration.js:128]: foreign keys should always be indexed.

**Review by fugufish — CHANGES_REQUESTED** [review]

File: migrations/20170524131128-initial_migration.js (hunk 2 above)
Comment (BrentFarese) [reply migrations/20170524131128-initial_migration.js:128]: Will add these and will be in the next commit.  Thanks!

**Review by BrentFarese — COMMENTED** [review]
Will add foreign key indexes in next commit.

**Pull request closed without merging**